src/geometry/CTiglPoint.cpp

```diff
@@ -177,4 +178,37 @@ void CTiglPoint::getMinMax(double & min, double & max) const
     }
 }
 
+bool CTiglPoint::operator==(const CTiglPoint& aPoint) const
+{
+    if (x == aPoint.x && y == aPoint.y && z == aPoint.z) {
```

Comment:
Never compare floating point types to equality!!! Always use a tolerance.

Reply:
Sure, done
commit: c2d9b9084f9cf34caa645b4b95a536bd62a6541b